src/D_EcobeeHouse1.json

```diff
@@ -35,10 +35,10 @@
     ],
     "inScene": "1",
     "DisplayStatus": {
-        "Service": "urn:upnp-org:serviceId:SwitchPower1",
-        "Variable": "Status",
-        "MinValue": "0",
-        "MaxValue": "1"
+        "Service": "urn:ecobee-com:serviceId:Ecobee1",
+        "Variable": "currentClimateRef",
+        "MinValue": "home",
+        "MaxValue": "smart1"
```

Comment:
Honestly, I started making modifications for my personal use. I just saw on the forum some people asking for this functionality and wanted to share what I did. I am no coder... I am an engineer. I just reversed engineered some of your code the laziest possible way. ha!
I did not know how to trigger the vacation concept. What I found was also that when I triggered "smart1", the ecobee UI would show actually a vacation icon... That was true when I first made these modifications. It seems to have been changed at some point.